octoprint_mrbeam/iobeam/airfilter.py

```diff
@@ -296,6 +307,11 @@ def pressure_drop_mainfilter(self):
             return mainfilter_pressure_avg - fan_pressure_avg
         return None
 
+    def exhaust_hose_is_blocked(self):
+        if self.model_id in self.AIRFILTER3_MODELS:
+            return self._pressure2 < self.AF3_PRESSURE2_MIN
+        return None
```

Comment:
Would it be more clear to return False?

Reply:
I would say no, as we don't know it for sure, it is a unknown state